group-actors/src/main/java/org/sunbird/actors/CreateGroupActor.java

```diff
@@ -24,7 +23,7 @@
 )
 public class CreateGroupActor extends BaseActor {
 
-  private GroupService groupService = GroupServiceImpl.getInstance();
+  private GroupService groupService = new GroupServiceImpl();
```

Comment:
Initing here will instantiate only for the number of actor instances - not for every API. So this is  buggy.

And secondly, we need to pass on the requestId MDC context - recall the ground-work did earlier. We must put to use here.